DEBUG 2022-03-11 16:03:13.462 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-11 16:03:13.463 UTC: 	 Running on: Linux lxplus782.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-11 16:03:13.463 UTC: 	 Executing command: 'status'
DEBUG 2022-03-11 16:03:13.478 UTC: 	 Found CRAB cache file /afs/cern.ch/user/b/bdanzi/.crab3
ERROR 2022-03-11 16:03:13.508 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/b/bdanzi/TRKPOG_last/CMSSW_10_6_18/src/MuonAnalysis/MuonAnalyzer/scripts/TagProbe/crab_TnP_ntuplizer_muon_Z_Run2018_MINIAOD_Run2018B
ERROR 2022-03-11 16:03:13.509 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/status.py", line 48, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 275, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 646, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 508, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/b/bdanzi/TRKPOG_last/CMSSW_10_6_18/src/MuonAnalysis/MuonAnalyzer/scripts/TagProbe/crab_TnP_ntuplizer_muon_Z_Run2018_MINIAOD_Run2018B
INFO 2022-03-11 16:03:13.531 UTC: 	 Log file is /afs/cern.ch/user/b/bdanzi/TRKPOG_last/CMSSW_10_6_18/src/MuonAnalysis/MuonAnalyzer/scripts/TagProbe/crab.log
